README.md

```diff
@@ -1 +1 @@
-# JS18_ProjectB_Group4
\ No newline at end of file
+# Skeleton project for Swagger
```

Comment:
You changed the title by accident.

Please add some information about your project, too: How to build, test and run